server/routes/checkAnswers.js

```diff
@@ -110,14 +118,16 @@ const typeOfHandcuffsUsed = handcuffsType => {
   return handcuffsType === 'ratchet' ? 'ratchet' : 'fixed bar'
 }
 
-const createNewIncidentObj = (offenderDetail, description, newIncident = {}, involvedStaff) => {
+const createNewIncidentObj = (offenderDetail, description, newIncident = {}, involvedStaff, incidentDate) => {
   return {
     offenderName: offenderDetail.displayName,
     offenderNumber: offenderDetail.offenderNo,
     location: description,
     forceType: newIncident.forceType,
     staffInvolved: convertArrayOfObjectsToStringUsingSpecifiedKey('name', involvedStaff),
     witnesses: convertArrayOfObjectsToStringUsingSpecifiedKey('name', newIncident.witnesses),
+    incidentDate: moment(incidentDate).format('DD/MM/YYYY'),
+    incidentTime: moment(incidentDate).format('HH:mm'),
```

Comment:
this could just be a single variable - the reason it's two on the other page is that we need to separate it for editing - as each part is input separately